### DISCUSSION TITLE
How to store unique timestamp

### DISCUSSION THREAD

**Discussion by rnurislom** [questdb/questdb#1737] [Ideas]
I have to store stock market data. When I receive update on candle, I should rewrite it, Insert creates duplicated timestamp. How to insert in not exist, but if exist to update it?

**Comment by pswu11** [comment]
Hi @rnurislom could you provide more context such as your table schema? 
Currently, QuestDB doesn't have `UPDATE` operation, but there are some workarounds depending on the use case.
